src/index.scss

```diff
@@ -12,10 +12,197 @@ code {
     monospace;
 }
 @import url("https://fonts.googleapis.com/css?family=Oswald&display=swap"); 
-
+@import url('https://rsms.me/inter/inter-ui.css');
 h1,h2,h3,h4 {
   color: rgba(33, 33, 33, 0.9);
 }
 p {
   color: rgba(33, 33, 33, 0.75);
-}
\ No newline at end of file
+}
+        
+/////////Profile Form Styling////////////////////////////////////////
+* {
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+$gradient: #F96B4C, #F23182;
+$radius: 10px;
+$visual-height: 68%;
+
+
+.demensions{
+  margin-top: 10rem;
+  width: 350px;
+  height: 603px;
+  background-image: radial-gradient(circle at 0% 0%, #373b52, #252736 51%, #1d1e26);
```

Comment:
I like how the dark mode looks, and I love the animation. As the whole app is light mode I think it would look as great with the light mode instead too. You can keep the gradient animation but change the background to a whitish color. Also, make sure you check the accessibility. For now, it shows a  lot of contrast errors and empty of missing labels. Make sure you add those too.